### PR TITLE
Add Notifications tab to Event page

### LINKED ISSUE #275: Add data table to Events for Notifications
On the event show page, there are currently two tabs. This would be to add an additional tab. The tab should have a data table showing notifications for this event. You should use the same JS code used for the Availability listing on one of the other tabs.

### PR DESCRIPTION
**Pull request by jeffhos**
Moved Notifications table into a partial and use it in the Event display.  Resolves #275.

### CODE REVIEW

**Comment by kgf** [comment]
Thanks for your help. I'll review this later today.

File: app/views/notifications/_notifications_table.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  <%# TODO: I need to allow info only Notifications %>
+  <tbody>
+    <% notifications.each do |notification| %>
+      <tr>
+        <td><%= notification.status %></td>
+        <td><%= notification.event ? (link_to notification.event.title, notification.event) : 'None' %></td>
+        <td><%= notification.scheduled_start_time %></td>
+        <td><%= notification.channels %></td>
+        <td><%= link_to 'Edit', edit_notification_path(notification) %></td>
```
Comment (kgf) [inline app/views/notifications/_notifications_table.html.erb]: Could you please add the CanCan permission here ? `if can edit @notification`

File: app/views/notifications/_notifications_table.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <th>Channels</th>
+      <th colspan="1"></th>
+    </tr>
+  </thead>
+
+  <%# TODO: I need to allow info only Notifications %>
+  <tbody>
+    <% notifications.each do |notification| %>
+      <tr>
+        <td><%= notification.status %></td>
```
Comment (kgf) [inline app/views/notifications/_notifications_table.html.erb:19]: We should add a column that is a truncation of the subject. Also, that column should be a link to the notification itself.

**Comment by kgf** [comment]
Looks pretty good, few small changes. 
If you had time to add some tests, that would also be great.

Thanks for your help,
Kevin

File: app/views/notifications/_notifications_table.html.erb (hunk 1 above)
Comment (kgf) [reply app/views/notifications/_notifications_table.html.erb]: Here is an example
 https://github.com/ReadyResponder/ReadyResponder/blob/development/app/views/people/index.html.erb#L39

File: app/views/notifications/_notifications_table.html.erb (hunk 2 above)
Comment (jeffhos) [reply app/views/notifications/_notifications_table.html.erb:19]: I don't see `subject` or anything that looks like that on `notification`.  What attribute did you want to display here?

Comment (kgf) [reply app/views/notifications/_notifications_table.html.erb:19]: Hey, aesome, it's not there.
It is here however
https://github.com/ReadyResponder/ReadyResponder/wiki/Notifications
Skip that then, I'll make a new issue for it directly, beyond the scope of this one.
Thanks !
Kevin

File: app/views/notifications/_notifications_table.html.erb (hunk 1 above)
Comment (kgf) [reply app/views/notifications/_notifications_table.html.erb]: If you could still get this one, I think we'd be ready to merge.

**Comment by kgf** [comment]
I don't quickly see why the tests have failed, but the failure is related to this pr

**Comment by jeffhos** [comment]
Yup, it has to do with adding the permission test to the notifications page... I'm looking at it right now.

**Comment by kgf** [comment]
Thanks for seeing this through to completion.

**Comment by jeffhos** [comment]
No problem... sorry it took a bit to get it sorted out.

**Pull request merged**